packages/forma-36-codemod/transforms/__testfixtures__/v4-radio.input.js

```diff
@@ -0,0 +1,51 @@
+import { RadioButton, RadioButtonField } from '@contentful/forma-36-react-components';
+
+<RadioButton id="Radio" labelText="some label text" name="option1" />;
+
+const isDisabled = false;
+const isChecked = true;
+<RadioButton
+  disabled={isDisabled}
+  checked={isChecked}
+  onChange={() => {}}
+  labelText="some label"
+  className="className" />;
+
+<RadioButton
+  type="radio"
+  id="radio-input"
+  disabled
+  value="value"
+  name="selected"
+  aria-disabled={true}
+  role="option" />;
+
+const selectedUnit = 'px';
+<RadioButtonField
+  className="radio-button"
+  id="unit-option-percent"
+  checked={selectedUnit === 'percent'}
+  labelText="percent"
+  value="percent"
+  onChange={() => {}}
+  labelIsLight
+/>;
+<RadioButtonField
```

Comment:
`RadioButtonField` is not transformed to `Radio` with `FormControl`?

Reply:
Only if needed, as you check on the `output` file, if it needs the validation message it get wrapper on the `FormControl`